src/help/util.ts

```diff
@@ -97,3 +98,32 @@ export function getHelpFlagAdditions(config: IConfig): string[] {
   const additionalHelpFlags = config.pjson.oclif.additionalHelpFlags ?? []
   return [...new Set([...helpFlags, ...additionalHelpFlags]).values()]
 }
+
+export function formatFlagDeprecationWarning(flag: string, opts: true | Deprecation): string {
```

Comment:
since there's only 1 word of difference, it could be consolidated?
formatDeprecationWarning(deprecationType: 'command' | 'flag', item: string, opts: true | Deprecation)